File: graphql-endpoint/src/main/java/tech/gklijs/graphql_endpoint/config/AxonConfig.java
package tech.gklijs.graphql_endpoint.config;

import com.thoughtworks.xstream.XStream;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

@Configuration
public class AxonConfig {

    @Bean
    public XStream xStream() {
        XStream xStream = new XStream();

        xStream.allowTypesByWildcard(new String[]{
                "java.util.**",
                "tech.gklijs.api.**"
        });
        return xStream;
    }
}

File: command-handler/src/main/kotlin/tech/gklijs/commandhandler/AxonConfig.kt
package tech.gklijs.commandhandler

import com.thoughtworks.xstream.XStream
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration


@Configuration
class AxonConfig {

    @Bean
    fun xStream(): XStream? {
        val xStream = XStream()
        xStream.allowTypesByWildcard(
            arrayOf(
                "tech.gklijs.api.**"
            )
        )
        return xStream
    }
}